src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -136,6 +147,11 @@ def create(self, bucket, uid):
             raise DashboardException(e, http_status_code=500, component='rgw')
 
     def set(self, bucket, bucket_id, uid):
+        bucket_tenant = bucket[:bucket.find('/')] if bucket.find('/') >= 0 else None
```

Comment:
Can you please add a comment here why it is necessary to strip the tenant in a special case. In some months nobody is remembering this.

Second, wouldn't it make sense to relocate the logic into ``strip_tenant_from_bucket_name``, too? This would make the ``set`` method more cleaner.

Reply:
Done.